src/journal/journald-server.c

```diff
@@ -2477,6 +2518,7 @@ int server_new(Server **ret) {
                 .ratelimit_burst = DEFAULT_RATE_LIMIT_BURST,
 
                 .forward_to_wall = true,
+                .forward_to_socket = { .sockaddr.sa.sa_family = AF_MAX },
```

Comment:
hmm, why use AF_MAX as marker for "unset" here? so far we always used AF_UNSPEC for this, afaik?